packages/peregrine/lib/talons/WishlistPage/useActionMenu.js

```diff
@@ -20,15 +21,17 @@ const dialogs = {
  */
 export const useActionMenu = (props = {}) => {
     const { id } = props;
-    const operations = mergeOperations(DEFAULT_OPERATIONS, props.operations);
+    const operations = mergeOperations(DEFAULT_OPERATIONS, props.operations, getWishlistConfigQuery);
```

Comment:
`getWishlistConfigQuery` should be before `props.operations` so it can still be easily overriden through targets